common/awf_vehicle_rviz_plugin/rviz_2d_overlay_msgs/msg/TrafficSignalArrayUI.msg

```diff
@@ -0,0 +1,2 @@
+builtin_interfaces/Time stamp
```

Comment:
This message is the same as `autoware_perception_msgs/TrafficSignalArray`. What is there a dedicated UI message for?

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/5957#issuecomment-1872148109
Qt's MOC has a reserved keyword `signals`, which would block this plugin from building when using the ` autoware_perception_msgs/TrafficSignalArray` so the easy work around was to just create a simple message converter to have `traffic_signals` as the keyword to access the data instead. @isamu-takagi

Reply:
How about trying `add_definitions(-DQT_NO_KEYWORDS)`? This flag disables `signal/slot` and you can replace them with `Q_SIGNALS/Q_SLOTS`.

Reply:
Well seems like that works, I tried a bunch of solutions and the only thing i came up with was the message converter, but this is much cleaner and won't need a message converter, so i'll fix up the changes accordingly without the need for the message conversion through the `autoware_auto_msgs_adapter` package. Thanks for that suggestion!